app/presenters/matrix_box_presenter.rb

```diff
@@ -50,7 +50,8 @@ def rss_log_to_presenter(rss_log, view)
       if target&.respond_to?(:thumb_image) && target&.thumb_image
         view.thumbnail(target.thumb_image,
                        link: { controller: target.show_controller,
-                               action: target.show_action, id: target.id })
+                               action: target.show_action, id: target.id },
+                       obs_data: obs_data_hash(target))
```

Comment:
This code allows the matrix box photo itself to serve as an AJAX link, e.g. to the naming form. Defaults to current link type, to minimize refactoring. As yet unimplemented.